### PR TITLE
Add Homebrew formula

### PR DESCRIPTION
**Pull request by adaszko**

### CODE REVIEW

**Review by cloudhead — COMMENTED** [review]
Nice. Can we put this in `/pkg/brew/radicle-tools.rb`?

File: HomebrewFormula/radicle-tools.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class RadicleTools < Formula
+  version "0.0.0"
+  desc "Radicle CLI tools"
+  homepage "https://github.com/radicle-dev/radicle-client-tools"
```
Comment (cloudhead) [inline HomebrewFormula/radicle-tools.rb:4]: Let's do `https://app.radicle.network/alt-clients.radicle.eth/radicle-client-tools`

File: HomebrewFormula/radicle-tools.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  url "https://github.com/radicle-dev/radicle-client-tools/releases/download/v#{version}/radicle-tools-x86_64-apple-darwin.tar.gz"
```
Comment (cloudhead) [inline HomebrewFormula/radicle-tools.rb:6]: We should start to think about how we could host binaries on radicle :thinking:

**Comment by adaszko** [comment]
> Nice. Can we put this in `/pkg/brew/radicle-tools.rb`?

Done ✅

File: HomebrewFormula/radicle-tools.rb (hunk 2 above)
Comment (adaszko) [reply HomebrewFormula/radicle-tools.rb]: Maybe.  That sounds like a task to be considered in its own right though, so maybe let's put it on a backlog?

Comment (cloudhead) [reply HomebrewFormula/radicle-tools.rb]: Aye, just thinking aloud

**Pull request closed without merging**